### PR TITLE
Update flagaction.sqf

### PR DESCRIPTION
**Pull request by Azylacus**
Clarified 2 messages

### CODE REVIEW

File: A3-Antistasi/flagaction.sqf [hunk 1]
```diff
@@ -99,7 +99,7 @@ switch _tipo do
 		fuego addAction ["Rest for 8 Hours", "skiptime.sqf",nil,0,false,true,"","(isPlayer _this) and (_this == stavros)",4];
 		fuego addAction ["Clear Nearby Forest", "clearForest.sqf",nil,0,false,true,"","(isPlayer _this) and (_this == stavros)",4];
 		fuego addAction ["On\Off Lamp", "onOffLamp.sqf",nil,0,false,true,"","(isPlayer _this)",4];
-		fuego addAction ["I hate the fog", "[10,0] remoteExec [""setFog"",2]",nil,0,false,true,"","(_this == stavros)",4];
+		fuego addAction ["Clear Fog", "[10,0] remoteExec [""setFog"",2]",nil,0,false,true,"","(_this == stavros)",4];
```
Comment (zalexki) [inline A3-Antistasi/flagaction.sqf]: i actually like the light tone of this.
@Barbolani77 ?

Comment (Barbolani77) [reply A3-Antistasi/flagaction.sqf]: Well, that's definetly not a lack of english but just a little joke...

Comment (zalexki) [reply A3-Antistasi/flagaction.sqf]: that's what i thought, @Azylacus can you remove this change ? thx

**Comment by Barbolani77** [comment]
removed

**Pull request closed without merging**